File: app/src/androidTest/java/com/ariasaproject/pararam/ExampleAndroidUnitTest.java
package com.ariasaproject.pararam;

import static org.junit.Assert.*;

import android.content.Context;
import android.support.test.InstrumentationRegistry;

import org.junit.Test;
import org.junit.runner.RunWith;
import androidx.test.ext.junit.runners.AndroidJUnit4;

@RunWith(AndroidJUnit4.class)
public class ExampleAndroidUnitTest {
    @Test
    public void useAppContext() throws Exception {
        Context appContext = InstrumentationRegistry.getTargetContext();
        assertEquals("com.ariasaproject.pararam", appContext.getPackageName());
    }
}

File: app/src/androidTest/kotlin/com/ariasaproject/pararam/ExampleAndroidUnitTest.kt
package com.ariasaproject.pararam

import static org.junit.Assert.*

import android.content.Context
import android.support.test.InstrumentationRegistry

import org.junit.Test
import org.junit.runner.RunWith
import androidx.test.ext.junit.runners.AndroidJUnit4

@RunWith(AndroidJUnit4.class)
class ExampleAndroidUnitTest {
    @Test fun useAppContext() {
        var appContext : Context
        appContext = InstrumentationRegistry.getTargetContext()
        assertEquals("com.ariasaproject.pararam", appContext.getPackageName())
    }
}
